configs/oracle.json

```diff
@@ -0,0 +1,10 @@
+{
+  "provider":"Oracle",
+  "CPU": "0.015",
+  "RAM": "0.002",
+  "GPU": "2.00",
+  "storage": "0.00005479452",
+  "defaultLBPrice": "0.0113",
+  "internetNetworkEgress": "0.0085",
+  "oraclePricingEndpoint":"https://apexapps.oracle.com/pls/apex/cetools/api/v1/products"
```

Comment:
Couple situations:
- You have custom pricing data you want to serve
- Your cluster is disconnected from the internet, or otherwise doesn't have access to this URI

Reply:
Does Oracle provide an endpoint with custom pricing in the same format or would the user set it up themselves, If it is Oracle provided is there authorization associated or would the expectation be that they are using a kind of workload identity auth?

For the air gapped example, is the thought that they would have an alternate endpoint available on a vpn or it would just be disable?

As an explanation, I am hammering on this point because that config file is a bit of a mess and I don't want to increase dependencies on it. In the short term I would adding that URL as a const

Reply:
1. Oracle has custom pricing, which requires authentication to access, as it is from your tenancy. 
2. For scenarios where the default URI is not accessible, users could provide their own endpoint.

Couple thoughts on the config file:
If the config file is too large, what would another means of providing external configurations?
OR, If there's another parameter we can reuse, that might also work.

Reply:
@anders-swanson sorry for the slow responses here, we've been on a holiday schedule:) 

Many providers have huge config files-- we tokenize them and scan them to extract pricing for just the resources you're running. We should definitely not block this PR on that, though.

Reply:
@Sean-Holcomb could we compromise by making this an environment variable?

Reply:
Hi Ajay,
I can move this to an envrionment variable. Is there any concern with overloading environment variables with config values?

Reply:
overloaded env variables with default is preferable to this config file having additional fields added to it.

Reply:
Sounds good! It may be some time until I can implement the update due to holiday break, but know that it will be done.

Reply:
hey @anders-swanson checking back in. Still on your radar? I'm hopeful this should be really quick!

Reply:
Hey, sorry for the delay, I've just returned from winter holidays. I plan to push out these changes tomorrow 🤞